tests/end-to-end/upgrade/main.go

```diff
@@ -99,6 +114,18 @@ func main() {
 	mfCli, err := mfClient.NewForConfig(k8sConfig)
 	fatalOnError(err, "while creating Microfrontends clientset")
 
+	dexConfig := dex.Config{}
```

Comment:
The `main.go` file should be responsible for register all test and run them. `dexConfig` is individual part for specific test and it should be sets inside concrete test. If everyone will add configuration of their test inside the main file it will be unreadable.

Reply:
No it is not. Main is responsible also for wiring dependency injection and dex.Config is somehting that will be used (or at least should) across many tests.